charts/memgraph-high-availability/values.yaml

```diff
@@ -36,6 +36,8 @@ memgraph:
       logPVCClassName: ""
       logPVC: true
       logPVCSize: "256Mi"
+  affinity:
+    enabled: false
```

Comment:
why not true by default?

Reply:
I set it up initially because the CI will fail, and you will need to pass the config each time in minikube, but I agree it should be `true` by default, will change this.